pkg/formatter/table.go

```diff
@@ -86,7 +92,19 @@ func CookbooksReportSummary(state *reporting.CookbooksReport) FormattedResult {
 	sortCookbookRecords(state.Records)
 
 	for _, record := range state.Records {
-		row := []string{record.Name, record.Version}
+
+		recordName, policyGroup, policy := "", "", ""
+		if state.Anonymize {
+			recordName = ShortFormat(record.Name)
```

Comment:
I wonder if we shouldn't `ShortFormat` all values? It's not an unreasonable length for these fieldst to truncate at,  especially since the output table is intended as a summary.

Reply:
Disregard, I just saw that the length is 10; I misread it as 19.